3rd-party/crsqlite-wasm/scripts/build-wasm.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+mkdir -p dist
+
+# This env variable is required by the crsql build process
+# (as it's, and I quote, "only meant to be built as a submodule of @vlcn.io/js")
+#
+# We're providing a dummy value here (this has nothing to do with the underlaying build, but here we are)
+export CRSQLITE_COMMIT_SHA="$(git rev-parse HEAD)"
+
+cd ../emsdk
+./emsdk install 3.1.45
+./emsdk activate 3.1.45
```

Comment:
Why do we need this specific `emcc` version?

Reply:
No idea from global point of view.

From in practice point of view:
- I can't remember the exact issue (the built binary wasn't compatible with the JS glue -- it would break at runtime)
- it was fixed by (selectively) copying the [script](https://github.com/vlcn-io/js/blob/main/build-wasm.sh) used in `@vlcn.io/js` package (called from the [root Makefile](https://github.com/vlcn-io/js/blob/main/Makefile))